docs/openxla.md

```diff
@@ -0,0 +1,25 @@
+# OpenXLA
+
+As of ***Jun 28***, PyTorch/XLA now ***pulls*** XLA from OpenXLA. 
```

Comment:
Oh, I meant to highlight certain words that I wanted to mention in the comment with the `***`. You can remove all the `***`s here. 

nit: Let's add the year, too -- June 28th, 2023

Reply:
thanks, updated